Resources/config/consumers.yml

```diff
@@ -1,6 +1,8 @@
 parameters:
   smartesb.consumers.queue.class: Smartbox\Integration\FrameworkBundle\Components\Queues\QueueConsumer
   smartesb.consumers.async_queue.class: Smartbox\Integration\FrameworkBundle\Components\Queues\AsyncQueueConsumer
+  smartesb.consumers.php_amqp_lib_async_queue.class: Smartbox\Integration\FrameworkBundle\Components\Queues\PhpAmqpLibAsyncConsumer
+  smartesb.consumers.php_amqp_lib_signal_queue.class: Smartbox\Integration\FrameworkBundle\Components\Queues\PhpAmqpSignalConsumer
```

Comment:
Declare the class as parameter is considered a symfony bad practise, see https://github.com/symfony/symfony/issues/11881

Reply:
Ok. done.